src/components/FileTreeNode/index.spec.tsx

```diff
@@ -674,6 +676,21 @@ describe(__filename, () => {
         createInternalMessage(messages[1]),
       ]);
     });
+
+    it('does not check universal messages', () => {
```

Comment:
What are "universal" messages? Same as "global" messages?

Reply:
They are bigger than global messages 🌎 🌟  *Global* was the name we used to describe [messages that apply to an entire file](https://github.com/mozilla/addons-code-manager/blob/6b83d050a04694ccfde099005d58f4a4fb832455/src/reducers/linter.tsx#L73). What I'm describing are messages that apply to the entire add-on (all files). I'm open to other naming ideas. I couldn't think of anything as concise as *universal*.